package.json

```diff
@@ -28,11 +36,19 @@
   },
   "homepage": "https://github.com/Edgar-Romero87/notes#readme",
   "dependencies": {
+
+    "@code-fellows/supergoose": "^1.0.11",
+    "minimist": "^1.2.5"
+  },
+  "devDependencies": {
+    "jest": "^26.5.3"
+
     "dotenv": "^8.2.0",
     "minimist": "^1.2.5",
     "mongoose": "^5.10.0"
   },
   "devDependencies": {
     "jest": "^26.4.1"
+
```

Comment:
move the `dotenv` and `mongoose` to below line 39. Then on line 45, or below `jest` add a " } ".Anything after that curly bracket go ahead and delete.

Reply:
Thank you, I will do that in a minute, I had some conflicts before my PR and didn't double-check my files.